.github/workflows/deploy-pr-preview.yml

```diff
@@ -2,6 +2,11 @@ name: Deploy to Firebase Hosting on PR
 
 on: pull_request
 
+# cancel in-progress workflows instances for the same PR
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
this will help to automatically cancel pending workflows for target PR if more recent job instance is started:
<img width="866" alt="image" src="https://user-images.githubusercontent.com/71640186/168590285-415d359d-1422-4873-b881-da0aac6aa65d.png">